utility/sort/sort.go

```diff
@@ -0,0 +1,30 @@
+package sort
+
+import (
+	"github.com/gojektech/proctor/proctord/jobs/metadata"
+	"sort"
+)
+
+type Sorter interface {
```

Comment:
we can do without an interface since it's not a generic one. 
Also, in any test, mocking this wouldn't serve much purpose as there are no 3rd party dependencies.

I think it makes sense to use only a function instead of defining a method on an object. The function can be named: `Procs(procList []metadata.Metadata)`

While invoking the function inside `sort` package, it'll be called hence: `sort.Procs(procList)`